makefiles/boards/sam0.inc.mk

```diff
@@ -24,8 +24,15 @@ DEBUG_ADAPTER ?= dap
 
 # EDBG can only be used with a compatible Atmel programmer
 ifeq ($(DEBUG_ADAPTER),dap)
-  # set this to either openocd or edbg
+  # set this to either openocd, jlink or edbg
   PROGRAMMER ?= edbg
+else ifeq ($(DEBUG_ADAPTER),jlink)
+  # only use JLinkExe if it's installed
+  ifneq (,$(shell command -v JLinkExe)))
```

Comment:
Does it work on macOS (I'm a total newbie with macOS)

Reply:
Given that I added it out of [considerations for portability](https://github.com/RIOT-OS/RIOT/pull/11725#discussion_r295396006), I would think so.